types/index.d.ts

```diff
@@ -286,6 +286,9 @@ export interface MessageInputProps {
   /** The component handling how the input is rendered */
   Input?: React.ElementType<MessageInputUIComponentProps>;
 
+  /** Change the SendButton component */
+  SendButton?: React.ElementType<any>;
```

Comment:
Instead of `any`, we should create an interface called `SendButtonProps` ... and assign it here!!